modules/loader/src/main/java/com/opengamma/strata/loader/fpml/FpmlDocumentParser.java

```diff
@@ -216,7 +272,7 @@ private static XmlElement findFpmlRoot(XmlElement root) {
         }
       }
     }
-    throw new FpmlParseException("Unable to find FpML root element");
+    return null;
```

Comment:
Is this a deliberate decision to avoid `Optional` for performance?

Reply:
There is no need to use `Optional` between two private methods of the same class. Sometimes it may make sense for other reasons, but didn't seem to do so here.